cms/djangoapps/contentstore/utils.py

```diff
@@ -455,3 +457,16 @@ def get_visibility_partition_info(xblock):
         "has_selected_groups": has_selected_groups,
         "selected_verified_partition_id": selected_verified_partition_id,
     }
+
+
+def is_self_paced(course):
```

Comment:
It needs utility because, we can not always use `course. self_paced` because in `create_xblock_info` method it does not have to be a course always ( sometimes library may be). Thus, `course` object may be None. Also, we still need to check on `SelfPacedConfiguration` is enabled or not.